src/OrbitGl/AsyncTrack.cpp

```diff
@@ -31,8 +31,9 @@ using orbit_client_protos::TimerInfo;
 using orbit_grpc_protos::InstrumentedFunction;
 
 AsyncTrack::AsyncTrack(CaptureViewElement* parent, TimeGraph* time_graph, TimeGraphLayout* layout,
-                       const std::string& name, OrbitApp* app, const CaptureData* capture_data)
-    : TimerTrack(parent, time_graph, layout, app, capture_data) {
+                       const std::string& name, OrbitApp* app, const CaptureData* capture_data,
+                       uint32_t indentation_level)
+    : TimerTrack(parent, time_graph, layout, app, capture_data, indentation_level) {
```

Comment:
Is indentation level the same as the level in the hierarchy of tracks? That is, in our case the main track has indentation_level 0, and the two subtracks each have indentation level 1 (and if they had subtracks they would have indentation level 2)?

Reply:
Exactly.